src/components/Tabs.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Tabs.css';
+
+export class Tabs extends React.Component {
+  state = {
+    index: 0,
+  }
+
+  hendleClickTab = (event) => {
+    this.setState({index: +event.target.value})
+  }
+
+  render() {
+    const { tabs } = this.props;
+    const { index } = this.state;
+
+    return (
+      <>
+        {tabs.map((tab, i) => (
+            <button
+              onClick={this.hendleClickTab}
+              className={i === index ? 'tab tab--active' : 'tab'}
+              value={i}
+              key={i}
```

Comment:
```suggestion
              key={tab.title}
```

it's a bad practice to use index as a key